recipes/rocksdb/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eu
+
+### Create Makefiles
+cmake -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DFAIL_ON_WARNINGS=ON \
+      -DPORTABLE=ON \
+      -DUSE_RTTI=ON \
+      -DWITH_GFLAGS=ON \
+      -DWITH_JEMALLOC=ON \
+      -DWITH_LZ4=OFF \
+      -DWITH_SNAPPY=ON \
+      -DWITH_TESTS=OFF \
+      -DWITH_TOOLS=OFF \
+      -DWITH_ZLIB=OFF \
+      -S . \
+      -B Build
+
+### Build
+cd Build
+make -j $CPU_COUNT
```

Comment:
I always use:
```cmake --build /path/to/builddir -- -j ${CPU_COUNT} ```
And:
```cmake --build /path/to/builddir -- install```

These both are generator agnostic.

Reply:
This is great! I wish I had known about this before. I switched to ninja, so I took the -j option out.